lib/iris/tests/unit/cube_coord_common/test_get_valid_standard_name.py

```diff
@@ -19,10 +19,26 @@ class Test(tests.IrisTest):
     def setUp(self):
         self.emsg = "'{}' is not a valid standard_name"
 
+    def test_none_is_valid(self):
```

Comment:
@jvegasbsc I'm going to be a tad fussy here, sorry... `None` is *valid* in the sense that no exception is raised by `get_valid_standard_name`, but obviously it's not an actual *valid* standard name.

Could we change the intent of this test name to be something like `test_none_passed_thru` instead?